### ISSUE TITLE
Fluid container

### ISSUE THREAD

**Issue by poopsplat** [billybobjobo/public_narrative#4]
We mentioned this a little earlier, that it would be idea for the responsive container to be fluid rather than jumping from one fixed width to another, but I realize this is all happening from within the bootstrap styles, so I think one way to get around it would be to just override the bootstrap styles in you `_global.scss` file like so: 

```
// Override bootstrap container fixed-width
.container {
	@media (min-width: $screen-sm-min) {
	  width: 100%;
	}
	@media (min-width: $screen-md-min) {
	  width: 100%;
	}
	@media (min-width: $screen-lg-min) {
		width: auto;
	  max-width: $container-lg;
	}
}
```

You'll definitely want to try that out and poke around the site pretty thoroughly to make sure it doesn't throw off anything else though!

**Comment by billybobjobo** [comment]
done locally.  need to put in a little time after rehearsal double checking
it doesn't break anything.

--

Bryant Smith
630.430.5328

**Comment by billybobjobo** [comment]
Ran into a snag in that I didn't realize base.php was making a wrapper (I keep tripping over all of sage's conveniences hahaha), but it's fixed now.

Bootstraps has a class that behaves fluidly, making the fix:

```css
.pn-wrapper {
	@extend							.container-fluid;
	max-width: 						1200px;
}
```

I had chosen a fixed container purposefully before because I was scared of all of the percentages.  But, I ended up using percentages as much as possible anyway.  As fas I can tell, I couldn't find any positioning issues arising after the above fix.
